client/dashboard/sites/backup-restore/error.tsx

```diff
@@ -0,0 +1,23 @@
+import { __experimentalHStack as HStack, Button } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import Notice from '../../components/notice';
+
+const SiteBackupRestoreSuccess = ( { onRetry }: { onRetry: () => void } ) => {
```

Comment:
We should change the name 🙂

Reply:
Yep, totally missed it :)
Thank you, addressed it on 0796acbd6bfed977b9fde527978c222345347e93